Traceback (most recent call last):
  File "d_cgcnn.py", line 120, in <module>
    f_params=f_params)
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/site-packages/skorch/net.py", line 1550, in load_params
    self.history = History.from_file(f_history)
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/site-packages/skorch/history.py", line 162, in from_file
    with open_file_like(f, 'r') as fp:
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/site-packages/skorch/utils.py", line 399, in open_file_like
    f = open(f, mode)
FileNotFoundError: [Errno 2] No such file or directory: './histories/valid_best_d30_history.json'
srun: error: cgpu14: task 0: Exited with exit code 1
srun: Terminating job step 363056.2
Traceback (most recent call last):
  File "d_cgcnn.py", line 120, in <module>
    f_params=f_params)
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/site-packages/skorch/net.py", line 1550, in load_params
    self.history = History.from_file(f_history)
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/site-packages/skorch/history.py", line 162, in from_file
    with open_file_like(f, 'r') as fp:
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/site-packages/skorch/utils.py", line 399, in open_file_like
    f = open(f, mode)
FileNotFoundError: [Errno 2] No such file or directory: './histories/valid_best_d25_history.json'
Traceback (most recent call last):
  File "d_cgcnn.py", line 120, in <module>
    f_params=f_params)
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/site-packages/skorch/net.py", line 1550, in load_params
    self.history = History.from_file(f_history)
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/site-packages/skorch/history.py", line 162, in from_file
    with open_file_like(f, 'r') as fp:
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/global/homes/k/ktran/miniconda3/envs/gaspy/lib/python3.6/site-packages/skorch/utils.py", line 399, in open_file_like
    f = open(f, mode)
FileNotFoundError: [Errno 2] No such file or directory: './histories/valid_best_d0_history.json'
srun: error: cgpu13: task 0: Exited with exit code 1
srun: Terminating job step 363056.0
srun: error: cgpu13: task 0: Exited with exit code 1
srun: Terminating job step 363056.1
